json-rpc/tests/integration_test.rs

```diff
@@ -548,12 +548,14 @@ fn create_test_cases() -> Vec<Test> {
                     vm_status,
                     json!({
                         "abort_code": 1288,
-                        "explanation": {
-                            "category": "LIMIT_EXCEEDED",
-                            "category_description": " A limit on an amount, e.g. a currency, is exceeded. Example: withdrawal of money after account limits window\n is exhausted.",
-                            "reason": "EINSUFFICIENT_BALANCE",
-                            "reason_description": " The account does not hold a large enough balance in the specified currency"
-                        },
+                        "explanation": serde_json::Value::Null,
+                        // TODO(philiphayes): re-enable when Move error explanations are published
```

Comment:
I don't think leave a todo is good idea

Reply:
Philip is leaving the TODOs in the code in order to drive comments during Review, I don't expect this PR to land with these TODOs still here.

Reply:
maybe use TBD, or discussion?

Reply:
re-enabled previous expected output now that explanations are working again